src/multi_vector_simulator/F2_autoreport.py

```diff
@@ -106,7 +106,9 @@ async def _print_pdf_from_chrome(path_pdf_report):
     await page.goto(
         "http://127.0.0.1:8050", {"waitUntil": "domcontentloaded", "timeout": 120000}
     )
-    await page.waitForSelector("#main-div")
+    await page.waitForSelector(
+        ".dash-cell", {"visible": "true",},
```

Comment:
this is the important part, you might want to explain what happens here, and maybe add a full entry for it in the changelog